src/components/App.js

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import CalculatorDisplay from './Calculator';
+import HomePage from './HomePage';
+import Nav from './Nav';
+import Quote from './Quote';
+import './App.css';
 
 // eslint-disable-next-line react/prefer-stateless-function
 class App extends React.Component {
```

Comment:
- [ ] [OPTIONAL] The app is producing a warning, kindly consider fixing this so your app is running without error/warnings

![Screenshot_20220809_204854](https://user-images.githubusercontent.com/25227022/183748374-b036643d-719e-4490-91e2-c7d44a41356c.png)

Reply:
Sure, It's solved now, thank you